bin/node/runtime/src/lib.rs

```diff
@@ -626,17 +630,15 @@ frame_election_provider_support::generate_solution_type!(
 		VoterIndex = u32,
 		TargetIndex = u16,
 		Accuracy = sp_runtime::PerU16,
-		MaxVoters = MaxElectingVoters,
+		MaxVoters = MaxNominations,
 	>(16)
 );
 
 parameter_types! {
+	pub ElectionBoundsMultiPhase: ElectionBounds = ElectionBoundsBuilder::new().voters_count(40_000).targets_count(10_000).build();
```

Comment:
Given that many ppl copy-pasta this code, let's use some smaller bounds, and add some comment about what they imply. This is a good resouce. This is a good resource: https://substrate.stackexchange.com/questions/2843/how-can-the-substrate-staking-pallet-be-used-in-a-parachain/3289#3289